android/app/src/main/java/org/chrome_on_device_ml/ml/TextClassification.java

```diff
@@ -0,0 +1,267 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome_on_device_ml.ml;
```

Comment:
Why is this class called TextClassification.java? Do we have QAanswer and TextClassification model both being run by the same client?

Reply:
The class exists but it's not running in the current app.

Reply:
How about removing it if not used?

Reply:
Done.